test/gpu/native/streamPrototype/stream.plot.gather.cuda.bash

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# sets 'datFile', 'logDir', 'experimentName', and 'runLog'
+source $CHPL_HOME/util/test/chplExperimentGatherUtils/boilerplate.bash $@
+
+sizes=( 1 2 4 8 16 32 64 128)
+
+# -----------------------------------------------------------------------------
+# Checkout and build CUDA code
+# -----------------------------------------------------------------------------
+rm -fr ./cuda-stream
+git clone https://github.com/bcumming/cuda-stream.git
+cd cuda-stream
+make
```

Comment:
The default arch is pretty old (`sm_35`) and some of the systems I've run on don't support that. Passing in `CHPL_GPU_ARCH` works for me though. I did this with:

```
make ARCH=$(printchplenv --all --internal 2>/dev/null | grep CHPL_GPU_ARCH | sed 's/^.*: //')
```